Lib/test/test_cmath.py

```diff
@@ -179,8 +176,6 @@ def test_infinity_and_nan_constants(self):
         self.assertEqual(repr(cmath.nan), "nan")
         self.assertEqual(repr(cmath.nanj), "nanj")
 
-    # TODO: RUSTPYTHON
-    @unittest.expectedFailure
```

Comment:
Is there any specific reason to remove this line

Reply:
This line is removed when the test passes.